eleventy.config.js

```diff
@@ -96,33 +96,17 @@ module.exports = function configure(eleventyConfig) {
       );
     }
 
-    let browserSyncConfig = {
-      callbacks: {
-        ready(err, bs) {
-          bs.addMiddleware('*', (req, res) => {
-            const content_404 = fs.readFileSync('./build/error.html');
-            // Provides the 404 content without redirect.
-            res.write(content_404);
-            // Add 404 http status code in request header.
-            res.writeHead(404);
-            res.end();
-          });
-        },
-      },
-    };
-
     if (process.env.USE_HTTPS === '1') {
-      browserSyncConfig = {
-        ...browserSyncConfig,
+      const serverOptions = {
         host: 'example.com',
```

Comment:
Do you remember if there was a reason `example.com` was used and not `localhost` or something using the reserved `.test` tld ? That makes the setup annoying, need to remember to add `127.0.0.1 example.com` to `/etc/hosts` ...

Reply:
yeah, `example.com` is hardcoded in gecko: https://searchfox.org/mozilla-central/rev/95ec620fda8ad6e4096cac1f1a9db8653669b31d/toolkit/mozapps/extensions/AddonManager.sys.mjs#59,61